pubnub.go

```diff
@@ -74,6 +75,27 @@ type PubNub struct {
 	ctx                  Context
 	cancel               func()
 	tokenManager         *TokenManager
+	cryptoModule         crypto.CryptoModule
+	previousCipherKey    string
+	previousIvFlag       bool
+}
+
+// TODO this needs to be tested
```

Comment:
How should we test getter function?

Reply:
As you can see it's kind of special getter function. It has actually a lot of things going on inside. Hopefully I figure out some not that awful way to test it

Reply:
So the only thing we can test is that we're getting `nil` in proper client configs. I think that there is no better way to do that ;/